packages/launchpad/src/lcdapi/lcdclient.spec.ts

```diff
@@ -195,11 +195,8 @@ describe("LcdClient", () => {
 
         {
           const recipient = makeRandomAddress();
-          const transferAmount = {
-            denom: "ucosm",
-            amount: "1234567",
-          };
-          const result = await client.sendTokens(recipient, [transferAmount]);
+          const transferAmount = coins(1234567, "ucosm");
```

Comment:
It does. The term "transfer" bothers me also because we have
- bank send messages,
- IBC transfer messages.

I suggest using `sendAmount` or simply `amount` to stay consistent with the terminology of `MsgSend`.

Should I include this here?

Reply:
I have no preference for one PR or two.

Reply:
Alright, done